sources/packages/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -224,6 +224,7 @@ export class StudentAESTController extends BaseController {
       firstName: eachStudent.user.firstName,
       lastName: eachStudent.user.lastName,
       birthDate: getISODateOnlyString(eachStudent.birthDate),
+      sin: eachStudent.sinValidation.sin,
```

Comment:
Is the below AC done?
![image](https://user-images.githubusercontent.com/61259237/178554833-eb2fc980-b4f9-49e9-81a6-0cd893d33166.png)

Reply:
Missed it, I will refactor.